src/WebJobs.Script/Properties/Resources.resx

```diff
@@ -176,4 +176,7 @@
   <data name="SasTokenExpiringInfoFormat" xml:space="preserve">
     <value>SAS token within '{1}' setting is set to expire in {0} days.</value>
   </data>
+  <data name="LinuxConsumptionRemoveTimeZone" xml:space="preserve">
+    <value>WEBSITE_TIME_ZONE and TZ are not currently supported on the Linux Consumption plan. Please remove these environment variables.</value>
```

Comment:
Can we tweak this slightly to `The environment variables 'WEBSITE_TIME_ZONE' and 'TZ' are not supported on this platform. For more information, see https://go.microsoft.com/fwlink/?linkid=2250165.`

That makes it a little more generic and lets us re-use this string if (for example) we need to apply it to Legion. Also gives us the link right in the message for use with the App Insights log.